Model/Payment_Method.php

```diff
@@ -124,6 +124,12 @@ public function getConfigFields()
                 'field'       => 'oxpayments__wdoxidee_httppass',
                 'title'       => Helper::translate('config_http_password'),
             ],
+            'testCredentials' => [
+                'type'        => 'button',
+                'onclick'     => 'wdTestPaymentMethodCredentials()',
+                'multilangid' => 'test_credentials',
```

Comment:
Please call this `title` and pass the translated string directly.

`'title' => Helper::translate('test_credentials'),`

Reply:
Good point! Named it 'text' now because otherwise it wouldn't work with the current logic whether the first column is displayed or not.